spaceShooter.py

```diff
@@ -46,6 +48,39 @@
 
 font_name = pygame.font.match_font('arial')
 
+def main_menu():
```

Comment:
How about reverting back to the main menu once again when all the lives get over?

Should we try this or we should try a different approach?